heavyai/_utils.py

```diff
@@ -139,24 +137,19 @@ def _parse_tdf_gpu(tdf):
         # columns
         pass
 
-    dtype = np.dtype(np.byte)
-    darr = cuda.devicearray.DeviceNDArray(
-        shape=ipc_buf.size,
-        strides=dtype.itemsize,
-        dtype=dtype,
-        gpu_data=ipc_buf.to_numba(),
-    )
-
-    reader = GpuArrowReader(schema, darr)
+    batch = pa.ipc.read_record_batch(ipc_buf.to_pybytes(), schema)
```

Comment:
Would `to_pybytes()` copy memory to host?

Reply:
@guilhermeleobas notice that `pa.ipc.read_record_batch(...)` results in a CPU batch.
For CUDA memory, we should use `pa.cuda.read_record_batch(ipc_buf, schema)`.

Reply:
Fixed